docs/getting_started.md

```diff
@@ -0,0 +1,31 @@
+# 📚 Overview
+
+Dozer is a Real-Time SQL Engine that connects to any of your data sources, transforms and stores the data in an embedded cache.
+
+**Core Features**
+
+* Simple to Use
+Dozer instantly generates fully indexed gRPC and REST APIs automatically. All you need is to configure a YAML file with your data source configuration and the APIs you want to deploy. 
+
+* Combine data from multiple sources
+Dozer can in real-time join data coming from multiple data sources powering advanced use cases. This can be done using the standard JOIN operator in SQL.
```

Comment:
Dozer supports a quasi-ANSI SQL. She would mention it. One of teh feature is JOIN but we also support aggregations